docs/README.md

```diff
@@ -34,10 +34,10 @@ The site can be viewed at http://localhost:1313/
 
 ## Generate configuration tables
 
-Configuration descriptions are auto generated from code. To trigger the generation you need to run:
+Configuration descriptions are auto generated from code. To trigger the generation you need to run in the project root:
 
 ```
-mvn -Pgenerate-config-docs install
+mvn -Pgenerate-config-docs install -DskipTests -Dcheckstyle.skip -Drat.skip -Dscalastyle.skip -Denforcer.skip=true -Dspotless.check.skip=true -Dskip.npm=true -DskipITs=true -Dmaven.javadoc.skip=true -Djapicmp.skip=true
```

Comment:
this might a bit too specific, who know what things we need to skip in the future. what we can suggest is `-DskipTests -Dfast` that should be enough for most use cases, no?

Reply:
yep let's just use `-dfast`